AirCasting/SessionViews/SessionCartFollowing.swift

```diff
@@ -32,7 +32,8 @@ extension SessionFollowing {
          set {
              #warning("❌This 'setter' is asynchronous!!!")
```

Comment:
Btw what do you think about this warning? Is it still relevant? Is it a problem that it is asynchronous?

Reply:
It can be a problem. If we set the value of this property and than shortly after (before database queue is processed) read from it. The value returned will still be an old one. It needs some analysis if that's a problem, but it should be somehow communicated to the user of this class (warning is not great)